tests/test_core.py

```diff
@@ -399,6 +399,84 @@ def test_parameter(self, spec):
                 == metadata["components"]["parameters"]["test_parameter"]
             )
 
+        spec.path(
```

Comment:
You could add a comment to explain the purpose of this test, and/or put this in another test method, like

```py
def test_parameter_duplicate(self, spec):
```

Reply:
Fixed in 3c01ee2e, moved tests in another method